corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -79,6 +82,19 @@ def test_location_restricted(self):
             accessible_locs
         )
 
+    def test_location_assigned_and_their_descendants(self):
+        self.web_user.add_to_assigned_locations(self.domain, self.locations['California'])
+        self.restrict_user_to_location(self.web_user)
```

Comment:
nit: I think this name is no longer clear, could be something like `remove_all_locations_access_for_user` or something.